### ISSUE TITLE
Crash keeps server from restarting

### ISSUE THREAD

**Issue by TheE** [TheE/MyWarp#45]
Follow-up from [here](http://dev.bukkit.org/bukkit-plugins/mywarp/?comment=425):
>There's a crash that blocks my server from starting properly. Could you please check? The crash log is at http://pastebin.com/j0YCsPmD

Relevant part:
```
[19:32:56] [Spigot Watchdog Thread/ERROR]: Current Thread: Server thread
[19:32:56] [Spigot Watchdog Thread/ERROR]:      PID: 17 | Suspended: false | Native: true | State: RUNNABLE
[19:32:56] [Spigot Watchdog Thread/ERROR]:      Stack:
[19:32:56] [Spigot Watchdog Thread/ERROR]:              org.sqlite.NativeDB.step(Native Method)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              org.sqlite.DB.execute(DB.java:328)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              org.sqlite.DB.executeUpdate(DB.java:363)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              org.sqlite.PrepStmt.executeUpdate(PrepStmt.java:85)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              me.taylorkelly.mywarp.internal.squirrelid.cache.SQLiteCache.executePut(SQLiteCache.java:138)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              me.taylorkelly.mywarp.internal.squirrelid.cache.SQLiteCache.putAll(SQLiteCache.java:117)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              me.taylorkelly.mywarp.internal.squirrelid.cache.AbstractProfileCache.put(AbstractProfileCache.java:36)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              me.taylorkelly.mywarp.internal.squirrelid.cache.SQLiteCache.put(SQLiteCache.java:47)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              me.taylorkelly.mywarp.bukkit.util.profile.SquirrelIdProfileService.onPlayerLogin(SquirrelIdProfileService.java:144)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              sun.reflect.GeneratedMethodAccessor181.invoke(Unknown Source)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              java.lang.reflect.Method.invoke(Method.java:497)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:301)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              net.minecraft.server.v1_8_R2.PlayerList.attemptLogin(PlayerList.java:453)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              net.minecraft.server.v1_8_R2.LoginListener.b(LoginListener.java:118)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              net.minecraft.server.v1_8_R2.LoginListener.c(LoginListener.java:54)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              net.minecraft.server.v1_8_R2.NetworkManager.a(NetworkManager.java:231)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              net.minecraft.server.v1_8_R2.ServerConnection.c(ServerConnection.java:148)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              net.minecraft.server.v1_8_R2.MinecraftServer.A(MinecraftServer.java:811)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              net.minecraft.server.v1_8_R2.DedicatedServer.A(DedicatedServer.java:368)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              net.minecraft.server.v1_8_R2.MinecraftServer.z(MinecraftServer.java:651)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              net.minecraft.server.v1_8_R2.MinecraftServer.run(MinecraftServer.java:554)
[19:32:56] [Spigot Watchdog Thread/ERROR]:              java.lang.Thread.run(Thread.java:745)
````

**Comment by TheE** [comment]
Does this happen in online or offline mode? 

If it happens in online mode, the issue might be caused by an issue within the native implementation of the SQLite driver. Updating the driver that CraftBukkit/Spigot bundles it relatively complex and involves removing the shaded one from the JAR manually.

**Comment by Minecraftly** [comment]
Hello,

it happens in offline mode because I use BungeeCord & Spigot.

Today, I think the MyWarp plugin auto deleted some warps (idk why, my users reported that their warps were removed accidentally), and the server runs for 24 hours without crashes. Do you know why? Thanks.

**Comment by TheE** [comment]
Problems with reappearing warps should be fixed in the latest dev build, see #43.

If you are using offline mode, I do not think that updating your SQLite driver will change anything. It is more likely that Bungecord somehow messed up a players unique ID causing the error on insert. Did the problem occur again within the last days?

**Comment by TheE** [comment]
Closing the issue as the problem appears to be resolved.
